### ISSUE TITLE
Not for Windows 10 Home

### ISSUE THREAD

**Issue by juanitogan** [eddex/aseprite-windows-docker-build#2]
Not an issue.  Just info.

You might note in your readme that this does not work on Windows 10 Home because Docker can only run WSL2 containers on Windows 10 Home.  The menu option to switch to Windows containers is still there on Windows 10 Home -- Docker just gets hung in starting mode when trying it.  Only in settings does Docker tell you that you can't turn off WSL2 containers.

Otherwise, in a Linux container (WSL2), it fails with this message when trying `docker build` (after the 7GB download):

```
failed to register layer: Error processing tar file(exit status 1): link /Files/Program Files/common files/Microsoft Shared/Ink/en-US/micaut.dll.mui /Files/Program Files (x86)/common files/Microsoft Shared/ink/en-US/micaut.dll.mui: no such file or directory
```

Anyhow, thanks for putting this together.

**Comment by eddex** [comment]
Odd. I use Windows 10 Pro so I can't test this myself but it could be that this is a limitation of Windows 10 Home.

It will for sure fail in a Linux container because there are many Windows specific tools and commands used in the build script.

Anyways, thanks for reporting! And there's always the option to either install everything needed for the build locally or simply buy the tool. 😄

**Comment by juanitogan** [comment]
Yes, it is definitely a limitation of Windows Home (according to Docker) and, thus, not a fixable issue.  Just a note for your readme.  Likely due to Home missing the VM features that Pro has.  But that is just a guess.  It is indeed odd that Home runs Linux stuff better than it does Windows stuff, in this case.

Anyhow, I just included the build message above in case anyone searches for it.  Even though Docker generally gets stuck in startup when trying to switch to Windows containers, it was also possible to confuse it into starting up anyway after requesting the switch.  And, thus, others might get the same message after thinking all was set correctly.  The settings page in Docker, however, tells the true story that you can only run WSL2 containers in Windows Home.

**Comment by juanitogan** [comment]
Oh, yeah, yet another option is LibreSprite.
